tutorials/navigation/navigation_using_navigationobstacles.rst

```diff
@@ -49,7 +49,7 @@ Ideally when an obstacle is moving the static vertices are removed and instead t
 Similar to agents the obstacles can make use of the ``avoidance_layers`` bitmask.
 All agents with a matching bit on their own avoidance mask will avoid the obstacle.
 
-Procedual obstacles
+Procedural obstacles
 ~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
~~~~~~~~~~~~~~~~~~~~
```

Missed this before, my bad. Underline length needs to be adjusted so it matches up with the section title.

Reply:
I'd missed too :) fixed